src/Foundatio.Aliyun/AliyunConnectionString.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Aliyun.OSS;
+
+namespace Foundatio {
+    internal class AliyunConnectionString {

```

Comment:
Perhaps this should be called AliyunStorageAccount (When reading the code I was curious what was being returned from parse), more inline with https://github.com/FoundatioFx/Foundatio.AzureStorage/blob/feature/logging/src/Foundatio.AzureStorage/Storage/AzureFileStorage.cs#L19